website/docs/guides/getting_started.html.markdown

```diff
@@ -280,4 +230,5 @@ If you ever need to make changes to your configuration, you can run `terraform a
 You can also run `terraform destroy` to tear down your resources if that's ever needed.
 
 [personal_api_key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key
```

Comment:
We seem to be calling this the use key now..  suggest we reference it correctly. 
```suggestion
[user_api_key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key
```